types/codemirror/codemirror-showhint.d.ts

```diff
@@ -1,6 +1,8 @@
 // Type definitions for CodeMirror
 // Project: https://github.com/marijnh/CodeMirror
-// Definitions by: jacqt <https://github.com/jacqt>, basarat <https://github.com/basarat>
+// Definitions by: jacqt <https://github.com/jacqt>
```

Comment:
Hi Ryan. Thanks for taking the time to check.
Could you be more specific about the issue? I was following the multiline format as suggested in the README: https://github.com/DefinitelyTyped/DefinitelyTyped#edit-an-existing-package